concrete/src/Workflow/BasicWorkflow.php

```diff
@@ -141,11 +142,11 @@ public function start(WorkflowProgress $wp)
                 $ui = UserInfo::getByID($req->getRequesterUserID());
 
                 // let's get all the people who are set to be notified on entry
-                $message = t(
+                $message = array(
```

Comment:
The problem is that if you wrap a t function around the string at that point you end up with emails in 2 languages. Is there not an additional way of having strings populated, because right now there is a lot of strings being run through the t function multiple times before a user will even see them.

If there isnt then I'll setup a protected function that will wrap the strings around a t function so they can be read by the translation system